### PR TITLE
Feature/jaxi support

### PR DESCRIPTION
**Pull request by JoostBaars**
This pull request adds support for the Jetson AGX Xavier Industrial module. The configuration files are added for the Industrial variant, required binary files, and the generation of the badpage.bin file (in tegra-bootfiles.bb). 
The generation of badpage.bin originates from the original flash.sh file from NVIDIA. 

I tested this on a Jetson AGX Xavier module (devkit) and the Jetson AGX Xavier Industrial module (on devkit carrier).

### CODE REVIEW

**Review by plyons72 — APPROVED** [review]
LGTM. Pretty much the same. We had a similar issue with writing /x01 to the badpage.bin file, and had originally wrapped it in a "bash -c" instead of a "python -c". Ultimately decided to just install the completed badpage.bin file in a bbappend instead of generating it on the fly, since this is not expected to change, and to keep it portable.

**Comment by JoostBaars** [comment]
Yeah, I added the binary first as well, but I think this is more neat, and it follows the approach of NVIDIA

**Comment by plyons72** [comment]
Thank you for adding this @JoostBaars! We've been waiting on internal approval to create a PR, but unfortunately these things take time. This will help a lot of folks going forward.

**Comment by ichergui** [comment]
@JoostBaars Could you please add `Signed-off-by` to your commit ? 

Please refer to our [CONTRIBUTING](https://github.com/OE4T/meta-tegra/blob/master/CONTRIBUTING.md) documentation

**Review by ichergui — CHANGES_REQUESTED** [review]
Please check my comments

File: recipes-bsp/tegra-binaries/tegra-bootfiles_35.3.1.bb
Comment (gsally-sarcos) [inline recipes-bsp/tegra-binaries/tegra-bootfiles_35.3.1.bb:86]: Is there a need to reuse an existing file?  The result is the same: a 4k file filled with zeros via lines 88 or 92.

Comment (JoostBaars) [reply recipes-bsp/tegra-binaries/tegra-bootfiles_35.3.1.bb:86]: This is the way NVIDIA implemented it in their flash tool. So I found it safer to just use their implementation, so the behavior between this layer and the NVIDIA flash script is limited

File: recipes-bsp/tegra-binaries/tegra-flashvars/jetson-agx-xavier-industrial/flashvars
Comment (JoostBaars) [reply recipes-bsp/tegra-binaries/tegra-flashvars/jetson-agx-xavier-industrial/flashvars]: Yeah, and I moved some of the others to the machine conf file as well

**Review by ichergui — APPROVED** [review]
LGTM 

Thanks @JoostBaars

**Review by gsally-sarcos — APPROVED** [review]

**Comment by gsally-sarcos** [comment]
lgtm, ty for your help!

**Comment by JoostBaars** [comment]
Thanks for reviewing everyone!

**Pull request merged**
